app/views/categories/index.html.erb

```diff
@@ -5,20 +5,22 @@
   </div>
```

Comment:
can you add a screenshot of the page?

Reply:
![2020-06-23 (3)](https://user-images.githubusercontent.com/64919284/85395400-f7a03a80-b547-11ea-8bd3-ad8c7e8d22e5.png)
![2020-06-23 (4)](https://user-images.githubusercontent.com/64919284/85395410-fa029480-b547-11ea-926d-86bdb5b25ab0.png)
Screenshots of the home page. Not the final one... Taking into consideration that it is suposed to be for mobile users.